templates/SilverStripe/Admin/Includes/LeftAndMain_MenuStatus.ss

```diff
@@ -3,8 +3,7 @@
         <a href="{$AdminURL}myprofile" class="cms-login-status__profile-link">
             <i class="font-icon-torso"></i>
             <span>
-                <%t SilverStripe\\Admin\\LeftAndMain.Hello 'Hi' %>
-                <% if $FirstName %>$FirstName<% else %>$Email<% end_if %>
+                <% if $FirstName && $Surname %>$FirstName $Surname<% else_if $FirstName %>$FirstName<% else %>$Email<% end_if %>
```

Comment:
oh, are we bringing back the full name?

Reply:
I know it was only removed but that was done to allow more space, I think removing the last name was one way to give more space to the first name but I thought it made more sense to actually adjust the CSS to provide more space to the name so there was less chance of using the ellipse. Removing the "Hi" also provide more space for the name.

I'm hoping this change makes sense, open to feedback but in my mind it's a good improvement :)

Reply:
cool, I had a similar idea when I merged the surname removal :)

lets put it back in for now then, there will need to be a framework pull request made to fix the behat tests for surname displayed - I'll create that now